src/algebra/group_power/basic.lean

```diff
@@ -20,20 +20,12 @@ Further lemmas can be found in `algebra.group_power.lemmas`.
 
 ## Notation
 
-The class `has_pow α β` provides the notation `a^b` for powers.
```

Comment:
Line 17 of the docstring above still claims that this file provides the definitions themselves. We should change that to "properties of the definitions", and explain where they're really defined and why